velero-plugins/common/shared.go

```diff
@@ -20,11 +19,25 @@ import (
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/runtime/schema"
 	"k8s.io/apimachinery/pkg/runtime/serializer"
-	"k8s.io/client-go/kubernetes"
+	"k8s.io/apimachinery/pkg/types"
 	"k8s.io/client-go/rest"
 )
+var (
+	registryInfo *string
+	serverVersion *serverVersionStruct
```

Comment:
Instead of saving these in global vars, why not add them to the `BackupPlugin` and `RestorePlugin` structs? Seems like that would be a much cleaner design.

Reply:
The reason is, these can be reused for the next backup if my understanding is right. Having them in BackupPlugin and RestorePlugin may result in higher memory usage.